runner/dl_checkpoints.sh

```diff
@@ -79,7 +79,42 @@ function download_all_models() {
 
     # Download live-video-to-video models.
     huggingface-cli download KBlueLeaf/kohaku-v2.1 --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
-    huggingface-cli download KwaiVGI/LivePortrait --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download stabilityai/sd-turbo --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download warmshao/FasterLivePortrait --local-dir models/FasterLivePortrait--checkpoints
+}
+
+function build_tensorrt_models() {
+    download_all_models
+
+    printf "\nBuilding TensorRT models...\n"
+
+    # Matrix of models and timesteps to compile StreamDiffusion TensorRT engines for.
+    MODELS="stabilityai/sd-turbo KBlueLeaf/kohaku-v2.1"
+    TIMESTEPS="3 4" # This is basically the supported sizes for the t_index_list
+    docker run --rm -it -v ./models:/models --gpus all \
+        livepeer/ai-runner:live-app-streamdiffusion \
+        bash -c "for model in $MODELS; do
+                    for timestep in $TIMESTEPS; do
+                        echo \"Building TensorRT engines for model=\$model timestep=\$timestep...\" && \
+                        python app/live/StreamDiffusionWrapper/build_tensorrt.py --model-id \$model --timesteps \$timestep
+                    done
+                done"
+
+    docker run --rm -it -v ./models:/models --gpus all \
+        livepeer/ai-runner:live-app-liveportrait \
```

Comment:
Will you need to first build docker images? Or will they be hosted in Docker Hub?

Reply:
They will be on docker hub! For now you have to build them though